input/_Head.cshtml

```diff
@@ -1,19 +1,12 @@
-        <link rel="apple-touch-icon" sizes="57x57" href="@Context.GetLink("/assets/img/apple-icon-57x57.png")" />
-        <link rel="apple-touch-icon" sizes="60x60" href="@Context.GetLink("/assets/img/apple-icon-60x60.png")" />
-        <link rel="apple-touch-icon" sizes="72x72" href="@Context.GetLink("/assets/img/apple-icon-72x72.png")" />
-        <link rel="apple-touch-icon" sizes="76x76" href="@Context.GetLink("/assets/img/apple-icon-76x76.png")" />
-        <link rel="apple-touch-icon" sizes="114x114" href="@Context.GetLink("/assets/img/apple-icon-114x114.png")" />
-        <link rel="apple-touch-icon" sizes="120x120" href="@Context.GetLink("/assets/img/apple-icon-120x120.png")" />
-        <link rel="apple-touch-icon" sizes="144x144" href="@Context.GetLink("/assets/img/apple-icon-144x144.png")" />
-        <link rel="apple-touch-icon" sizes="152x152" href="@Context.GetLink("/assets/img/apple-icon-152x152.png")" />
-        <link rel="apple-touch-icon" sizes="180x180" href="@Context.GetLink("/assets/img/apple-icon-180x180.png")" />
-        <link rel="icon" type="image/png" sizes="192x192" href="@Context.GetLink("/assets/img/android-icon-192x192.png")" />
-        <link rel="icon" type="image/png" sizes="32x32" href="@Context.GetLink("/assets/img/favicon-32x32.png")" />
-        <link rel="icon" type="image/png" sizes="96x96" href="@Context.GetLink("/assets/img/favicon-96x96.png")" />
-        <link rel="icon" type="image/png" sizes="16x16" href="@Context.GetLink("/assets/img/favicon-16x16.png")" />
-        <meta name="msapplication-TileColor" content="#ffffff" />
-        <meta name="msapplication-TileImage" content="@Context.GetLink("/assets/img/ms-icon-144x144.png")" />
-        <meta name="theme-color" content="#ffffff" />
+        <link rel="apple-touch-icon" sizes="180x180" href="@Context.GetLink("/assets/img/favicons/apple-touch-icon.png")">
+        <link rel="icon" type="image/png" sizes="32x32" href="@Context.GetLink("/assets/img/favicons/favicon-32x32.png")">
+        <link rel="icon" type="image/png" sizes="16x16" href="@Context.GetLink("/assets/img/favicons/favicon-16x16.png")">
+        <link rel="manifest" href="@Context.GetLink("/assets/img/favicons/manifest.json")"/>
+        <link rel="shortcut icon" type="image/x-icon" href="@Context.GetLink("/assets/img/favicons/favicon.ico")"/>
+        <link rel="mask-icon" href="@Context.GetLink("/assets/img/favicons/safari-pinned-tab.svg")" color="#ffffff">
+        <meta name="msapplication-config" content="@Context.GetLink("/assets/img/favicons/browserconfig.xml")" />
+        <meta name="msapplication-TileColor" content="#ffffff">
+        <meta name="theme-color" content="#ffffff">
```

Comment:
Why remove `msapplication-TileImage`  and different dimensions of `apple-touch-icon`?

Reply:
this is the best practice that has been generated by this service https://realfavicongenerator.net/
Also, such a number of icons is unnecessary. For IE we use browserconfig.xml